autoload/go/rename.vim

```diff
@@ -52,7 +52,12 @@ function! go#rename#Rename(bang, ...) abort
     return
   endif
 
-  let [l:out, l:err] = go#util#ExecInDir(l:cmd)
+  let l:wd = go#util#ModuleRoot()
+  if l:wd == -1
+    let l:wd = getcwd()
```

Comment:
`l:wd` will need to be set to the directory of the current buffer here for parity with current behavior.